tasks/cfme-repositories.yml

```diff
@@ -0,0 +1,8 @@
+- name: CFME | Disable All Repositories
```

Comment:
this is unrelated to the new playbook and is rather a fix for an existing playbook so maybe just submit as a separate pull request.